File: Frontend/trackr-paciente/src/app/shared/models/examen/examen-reactivo.ts
import { Time } from "@angular/common";
import { Respuesta } from "./respuesta";
export class ExamenReactivo{

    public idExamenReactivo: number;
    public idExamen: number;
    public idReactivo: number;
    public resultado: boolean;
    public respuestaAlumno: string;
    public respuestaValor : number;
    public fechaAlta: Date;
    public estatus: boolean;

    public asignatura: string;
    public clave: string;
    public pregunta: string;
    public imagenBase64: string;
    public respuestas: Respuesta[];
    public necesitaRevision: boolean;

    constructor() {}
}

File: Backend/TrackrAPI/TrackrAPI/Models/ExamenReactivo.cs
using System;
using System.Collections.Generic;

namespace TrackrAPI.Models
{
    public partial class ExamenReactivo
    {
        public int IdExamenReactivo { get; set; }
        public int IdExamen { get; set; }
        public int IdReactivo { get; set; }
        public bool? Resultado { get; set; }
        public string? RespuestaAlumno { get; set; }
        public DateTime? FechaAlta { get; set; }
        public bool? Estatus { get; set; }
        public int? RespuestaValor { get; set; }

        public virtual Examen IdExamenNavigation { get; set; } = null!;
        public virtual Reactivo IdReactivoNavigation { get; set; } = null!;
    }
}
